Software/Python/I2C_mutex.py

```diff
@@ -1,27 +1,56 @@
 import fcntl
```

Comment:
This is the First one I'm worried about.  I will defer to @CleoQc  on this one though.  We might also have @mattallen37  look at it.